manifest.json

```diff
@@ -6,7 +6,7 @@
   "version": "1.5.1",
   "homepage_url": "https://github.com/UNINETT/SAML-tracer",
   "icons": {
-    "48": "src/resources/images/saml-as-square.png"
+    "48": "src/resources/images/icon.svg"
```

Comment:
Hi Thijs! 👋🙂

While [Firefox can handle SVGs](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/icons#SVG), it seems that [Chrome requires PNGs](https://developer.chrome.com/apps/manifest/icons) for the icons.
Could you please create some and add them?

![image](https://user-images.githubusercontent.com/17160647/61183754-57f2e080-a645-11e9-8e7e-45132ef265af.png)

Reply:
That is really quite backward. Chrome claims that they can be "be in any format supported by WebKit" but this is apparently not true. Pity because we cannot use this really elegant solution. Anyway. I've replaced them with pre-scaled pngs.